SoftwareGuide/Latex/Introduction/Installation.tex

```diff
@@ -17,7 +17,7 @@ \chapter{Configuring and Building ITK}
 systems, compilers, and hardware platforms including Microsoft Windows, Linux on
 various architectures, UNIX, macOS, and Mingw-w64. Dedicated community members and
 Kitware are committed to providing long-term support of the most prevalent
-development environments (Visual Studio, macOS, and Linux) for building ITK:
+development environments (e.g. Clang and Visual Studio) for building ITK.
```

Comment:
In commit message "Tke advantage" -> "Take advantage"

Reply:
Fixed in 94e1127. Thanks @dzenanz !